augur/tasks/github/util/github_api_key_handler.py

```diff
@@ -130,6 +133,18 @@ def get_api_keys(self) -> List[str]:
         if not valid_keys:
             raise NoValidKeysError("No valid github api keys found in the config or worker oauth table")
 
+
+        # shuffling the keys so not all processes get the same keys in the same order
+        valid_now = valid_keys
+        try: 
+            self.logger.debug(f'valid keys before shuffle: {valid_keys}')
+            valid_keys = random.sample(valid_keys, len(valid_keys))
+            self.logger.debug(f'valid keys AFTER shuffle: {valid_keys}')
+        except Exception as e: 
+            self.logger.debug(f'{e}')
+            valid_keys = valid_now
```

Comment:
This only gets executed the first time the GithubApiKeyHandler is created after that the keys will be stored in Redis so this function will return in the `if redis_keys` block.